packages/e2e-tests/specs/block-deletion.test.js

```diff
@@ -108,4 +109,29 @@ describe( 'block deletion -', () => {
 			expect( await getEditedPostContent() ).toMatchSnapshot();
 		} );
 	} );
+
+	describe( 'deleting all paragraphs', () => {
+		it( 'results in the default block getting selected', async () => {
+			// Add a third paragraph for this test.
+			await page.keyboard.type( 'Third paragraph' );
+
+			// Move the mouse to show the block toolbar
```

Comment:
Pressing escape is an easier way to break out of the typing mode.

```js
await page.keyboard.press( 'Escape' );
```

I've mentioned this quite a few times recently. I think we should remove all the other instances of `page.mouse.move`, since the precedent is I think causing people to copy the pattern.